nginx.conf

```diff
@@ -0,0 +1,35 @@
+worker_processes auto;
+pid /var/run/nginx.pid;
+
+events {
+    worker_connections 10000;
+    multi_accept on;
+    use epoll;
+}
+
+http {
+    sendfile on;
+    tcp_nopush on;
+    tcp_nodelay on;
+    keepalive_timeout 65;
+    types_hash_max_size 2048;
+    client_max_body_size 20m;
+
+    log_format  main  '$remote_addr $host $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" $upstream_cache_status' ;
+    access_log /dev/stdout main;
+    error_log /dev/stderr;
+
+    gzip on;
+    gzip_disable "msie6";
+    gzip_proxied any;
+
+    real_ip_header X-Forwarded-For;
+    real_ip_recursive on;
+    set_real_ip_from 10.0.0.0/8;
+    set_real_ip_from 172.30.0.0/16;
```

Comment:
what address ranges are these trusted proxies ?

Reply:
These are two private IP ranges. The first is the internal EC2/Azure network and the second is used by Docker if I remember correctly.